ulid/__init__.py

```diff
@@ -297,7 +308,9 @@ def __get_pydantic_core_schema__(cls, source: Any, handler: GetCoreSchemaHandler
             core_schema.union_schema([
                 core_schema.is_instance_schema(ULID),
                 core_schema.no_info_plain_validator_function(ULID),
-                core_schema.str_schema(pattern=r"[A-Z0-9]{26}", min_length=26, max_length=26),
+                core_schema.str_schema(
+                    pattern=rf"[{base32.ENCODE}]{{26}}", min_length=26, max_length=26
```

Comment:
There is a test that would need to be adapted. Good catch with the `I`, though.

Reply:
Ok, I missed this issue and I'll follow up on a fix